easybuild/easyconfigs/t/taco/taco-0.5.1-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'taco'
+version = '0.5.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://tacorna.github.io'
+description = """Multi-sample transcriptome assembly from RNA-Seq"""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/tacorna/taco/archive']
+
+dependencies = [
+    ('Python', '2.7.12'),
+]
+
+postinstallcmds = ["mv %(installdir)s/bin/taco_run.py %(installdir)s/bin/taco_run"]
```

Comment:
@wpoely86 what made you rename this to `taco_run` rather than `taco`, which seems to be the usual main command (see 'Getting started' at https://tacorna.github.io/)?